lib/portlayer/exec/container.go

```diff
@@ -591,7 +594,7 @@ func (c *Container) Remove(ctx context.Context, sess *session.Session) error {
 		}
 		switch f.Fault().(type) {
 		case *types.InvalidState:
-			log.Warnf("container VM is in invalid state, unregistering")
+			log.Warnf("Container %s is in invalid state, unregistering", c.ExecConfig.ID)
 			if err := c.vm.Unregister(ctx); err != nil {
 				log.Errorf("Error while attempting to unregister container VM: %s", err)
```

Comment:
Might spew the error here similar to vic-machine delete? It's also possible to happen here in theory. :)